src/app/dynamic-forms/models/cinchy-form.model.ts

```diff
@@ -812,11 +812,10 @@ export class Form {
     fields.push("[Cinchy ID]");
 
     if (this.isChild) {
-      
       const defaultWhere = `where t.${this.childFormLinkId} = @parentCinchyIdMatch and t.[Deleted] IS NULL`;
 
-      const whereConditionWithFilter = this.childFormFilter ? 
-      `${defaultWhere} AND (${this.childFormFilter})` : defaultWhere;
+      const whereConditionWithFilter = this.childFormFilter ?
+        `${defaultWhere} AND (${this.childFormFilter})` : defaultWhere;
```

Comment:
This should either be all on the same line or (preferably) three lines, where each segment of the ternary statement has its own line for maximum clarity.

Reply:
done